### PR TITLE
Chapter1 homework 1and2

### PR DESCRIPTION
**Pull request by joshuajz**
Continuation of https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/282 with the fixes required & the proper pull request (I hope).

I've closed the previous pull request.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,56 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Person 1:
+            // Gathering Input:
+            // Name
+            Console.WriteLine("Enter Name: ");
+            string nm1 = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:13]: Avoid naming variables with abreviations, unless those abereviations are well known, common knowledge.

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,56 @@ class Program
...
+            // Weight
+            Console.WriteLine("Enter Weight: ");
+            int weight1 = int.Parse(Console.ReadLine());
+
+            // Height
+            Console.WriteLine("Enter Height: ");
+            float height1 = float.Parse(Console.ReadLine());
+
+            // Calculating BMI
+            float bmi1 = weight1 / (height1 * height1) * 10000;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:28]: For example, bmi is a well-known abbreviation and full name (Body-Mass-Index) is not needed in this context. So this is correct.

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,56 @@ class Program
...
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:29]: No need to put 2 extra blank lines, 1 line is going to be enough.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,56 @@ class Program
...
+            Console.WriteLine("Enter Height: ");
+            float height2 = float.Parse(Console.ReadLine());
+
+            // Calculating BMI
+            float bmi2 = weight1 / (height1 * height1) * 10000;
+
+
+            // Outputting Results:
+            // Person 1:
+            Console.WriteLine($"My name is {nm1} and I am {age1} years old, my weight is {weight1} kg, I'm {height1} meters tall and my BMI is {bmi1}");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:55]: This is fine, as it was not mentioned when it should be calculated. However, you should try and consume the code as soon as you code:
For person one, output results right after reading person1 info,
For person two, output the results right after reading person2 info.

**Comment by joshuajz** [comment]
I've finished the above changes :)

**Pull request closed without merging**